docs/docs/community/create-provider.md

```diff
@@ -73,22 +90,21 @@ export class ExampleProviderEmailProvider implements IEmailProvider {
 
   async sendMessage(options: IEmailOptions): Promise<ISendMessageSuccessResponse> {
     return {
-      id: 'PLACEHOLDER',
-      date: 'PLACEHOLDER',
-    };
+      id: 'id_returned_by_provider',
+      date: 'current_time'
   }
 }
 ```
 
-Template test case for `emailProvider`.
+Template test case for `example-provider`.
 
-```ts
-import { ExampleProviderEmailProvider } from './exampleProvider.provider';
+```ts title='providers/example-provider/src/lib/example-provider.provider.sepc.ts'
+import { ExampleProviderEmailProvider } from './example-provider.provider';
 
-test('should trigger exampleProvider library correctly', async () => {});
+test('should trigger example-provider library correctly', async () => {});
 ```
 
-### Email Provider
+### Sample Email Provider (SendGrid)
```

Comment:
Does this make sense now?

Reply:
Not much sense, just a example that sendMessage function will be changed, 
I think in last steps I should mention that checkout sendgrid provider to know other details?

Reply:
That could be a good option.